src/test/EigenPod.t.sol

```diff
@@ -614,6 +664,8 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         eigenPodManager.stake{value: 12 ether}(_pubkey, _signature, _depositDataRoot);
         
         // successful call
+        cheats.expectEmit(true, false, false, false);
```

Comment:
no address check?

Reply:
pretty sure I just copy-pasted the existing call to `cheats.expectEmit`. I'll fix it everywhere

Reply:
fixed in https://github.com/Layr-Labs/eigenlayer-contracts/pull/44/commits/b9be2206e32fd06a43963152bc5e128eb6138a17